recipe/meta.yaml

```diff
@@ -18,61 +18,66 @@ build:
 
 requirements:
   host:
-    - jupyter-packaging >=0.9,<1.0
+    - nodejs >=16
+    - hatch-jupyter-builder >=0.8.1
+    - hatchling >=1.11
     - pip
-    - python >=3.7
+    - python >=3.8
   run:
-    - anyio >=3.1.0
+    - anyio >=3.1.0,<4
     - argon2-cffi
     - jinja2
-    - jupyter_client >=6.1.12
-    - jupyter_core >=4.7.0
+    - jupyter_client >=7.4.4
+    - jupyter_core >=4.12,!=5.0
```

Comment:
@conda-forge/jupyter_server are we good with this?

Reply:
:+1: